GeoNewsFinder4/App.js

```diff
@@ -1,13 +1,26 @@
 import { StatusBar } from 'expo-status-bar';
 import { StyleSheet, Text, View } from 'react-native';
+import MapView from 'react-native-maps';
+import { WebView } from 'react-native-webview';
```

Comment:
Unused import -- let's remove this for now as we implement the web view in a separate PR. Same goes for StatusBar import above.